mlptrain/potentials/mace/mace.py

```diff
@@ -388,442 +92,147 @@ def valid_fraction(self) -> float:
-        if self.args.compute_avg_num_neighbors:
-            return modules.compute_avg_num_neighbors(self.train_loader)
-        else:
-            return self.args.avg_num_neighbors
+    def args(self) -> 'argparse.Namespace':
+        """Namespace containing mostly default MACE parameters"""
 
-    @property
-    def output_args(self) -> Dict:
-        """Dictionary containing required outputs"""
-        return {
-            'energy': True,
-            'forces': True,
-            'virials': False,
-            'stress': False,
-            'dipoles': False,
-        }
+        args_list = [
+            '--name',
+            self.name,
+            '--max_L',
+            str(Config.mace_params['max_L']),
+            '--train_file',
+            f'{self.name}_data.xyz',
+            '--valid_fraction',
+            str(self.valid_fraction),
+            '--energy_weight',
+            str(Config.mace_params['energy_weight']),
+            '--forces_weight',
+            str(Config.mace_params['forces_weight']),
+            '--config_type_weights',
+            str(Config.mace_params['config_type_weights']),
+            '--E0s',
+            str(self.get_E0s),
+            '--model',
+            str(Config.mace_params['model']),
+            '--hidden_irreps',
+            str(Config.mace_params['hidden_irreps']),
+            '--r_max',
+            str(Config.mace_params['r_max']),
+            '--lr',
+            str(0.001),
+            '--scaling',
+            'rms_forces_scaling',
+            '--batch_size',
+            str(self.batch_size),
+            '--valid_batch_size',
+            str(self.batch_size),
+            '--max_num_epochs',
+            str(Config.mace_params['max_num_epochs']),
+            '--error_table',
+            str(Config.mace_params['error_table']),
+            '--loss',
+            str(Config.mace_params['loss']),
+            '--correlation',
+            str(Config.mace_params['correlation']),
+            '--scheduler_patience',
+            str(20),
+            '--patience',
+            str(50),
+            '--start_swa',
+            str(Config.mace_params['start_swa']),
+            '--swa',
+            '--ema',
+            '--ema_decay',
+            str(0.999),
+            '--device',
+            str(Config.mace_params['device']),
+            '--default_dtype',
+            str(Config.mace_params['dtype']),
+            '--seed',
+            str(345),
+            '--energy_key',
+            'energy',
+            '--forces_key',
+            'forces',
+            '--num_workers',
+            str(Config.mace_params['num_workers']),
+        ]
+
+        if self.foundation is not None:
+            args_list.append('--foundation_model')
+            args_list.append(f'{self.foundation}')
+
+        if Config.mace_params['save_cpu'] is True:
+            args_list.append('--save_cpu')
+
+        if Config.mace_params['restart_latest'] is True:
```

Comment:
is True for a boolean is redundant (except for readability I suppose...)
```suggestion
        if Config.mace_params['restart_latest']:
```